### ISSUE TITLE
Unable to upload an app form after changing its `.properties.json`.

### ISSUE THREAD

**Issue by binokaryg** [medic/medic-conf#301]
Unable to upload an app form after changing its `.properties.json`. Change not detected.
With medic-conf 3.1.0, when uploading a form, it first checks whether there is any change in the form before uploading.

However, it does not detect if there is a change only in `form.properties.json`.

Steps to replicate:
- Upload an app form using medic-conf@3.1.0.
- Change something in `.properties.json` file of that form.
- Try converting and uploading the form again.

```
INFO Form /mnt/d/MedProject/medic-projects/nepal-healthform/forms/app/declaration.xml not uploaded, no changes        
INFO upload-app-forms complete. 
INFO All actions completed. 
```

**Comment by kennsippell** [comment]
Same for any files in the form's `media` folder!

**Comment by garethbowen** [comment]
Ready for AT in `301-check-properties-too`

This branch fixes: #271 #300 #301 #303 #306 so AT them at the same time.

**Comment by ngaruko** [comment]
LGTM .
Also double-checked with @binokaryg !
